src/bindings/python/tests/test_onnx/test_ops_convpool.py

```diff
@@ -11,10 +12,10 @@
 from tests.test_onnx.utils import get_node_model, import_onnx_model, run_model, run_node
 
 
-@pytest.fixture
+@pytest.fixture()
```

Comment:
what kind of error it was?
Personally, I don't like `()` after decorators.
I'd ignore this mark.

@jiwaszki what do you think?

Reply:
It was `PT001 use @pytest.fixture() over @pytest.fixture`

Reply:
It actually depends on what the decorator is doing. As decorators themselves could have arguments and return new customized decorator from them.

As far as I know this is a case for `@pytest.fixture()`, looking at implementation:
https://docs.pytest.org/en/6.2.x/_modules/_pytest/fixtures.html#fixture

Citing:
> This decorator can be used, with or without parameters, to define a fixture function.

You may see that empty `()` indicates there is no fixture function passed and following "The Zen of Python" second rule `Explicit is better than implicit.` empty brackets are more "in-style".